### PR TITLE
createMETSRights: namespace PREMIS elements

### PR DESCRIPTION
**Pull request by mistydemeo**
Fixes #7210.

### CODE REVIEW

**Comment by Hwesta** [comment]
If the METS still validates, :+1:

**Comment by mistydemeo** [comment]
Updated to also fix Trim and Dspace.

File: src/MCPClient/lib/clientScripts/archivematicaCreateMETSTrim.py [hunk 1]
```diff
@@ -93,7 +93,7 @@ def getTrimFileDmdSec(baseDirectoryPath, fileGroupIdentifier, fileUUID):
         return None
     for row in rows:
         xmlFilePath = row[0].replace('%SIPDirectory%', baseDirectoryPath, 1)
-        dublincore = etree.SubElement(xmlData, "dublincore", attrib=None, nsmap={None: ns.dctermsNS})
+        dublincore = etree.SubElement(xmlData, ns.dctermsBNS + "dublincore", attrib=None, nsmap={"ns": ns.dctermsNS})
```
Comment (Hwesta) [inline src/MCPClient/lib/clientScripts/archivematicaCreateMETSTrim.py]: Is `"ns"` supposed to be `"dc"` or  `"dcterms"`?

Comment (mistydemeo) [inline src/MCPClient/lib/clientScripts/archivematicaCreateMETSTrim.py]: `"dc"`, good catch. Fixed.

**Comment by Hwesta** [comment]
:+1:

**Pull request merged**